src/scripts/match-mondo-sources-all-lexical.py

```diff
@@ -85,7 +85,8 @@ def main(verbose: int, quiet: bool):
 def run(input: str, config: str, rules: str, rejects: str, output: str):
     # Implemented `meta` param in `lexical_index_to_sssom`
 
-    meta = get_metadata_and_prefix_map(config)
+    #meta = get_metadata_and_prefix_map(config)
+    meta = None
```

Comment:
## #394 redundancy
The main difference is that #397 does passes `meta=None` to `lexical_index_to_sssom()`, whereas #394 passes a `Converter`. Supposedly #397 approach is fine.

Everything else in that PR is now cosmetic, so i can merge just those changes afterwards or ignore and close.

My instinct is to feel a little irked that I spent a lot of work and found a way to get the pipeline working without throwing the `oio` err (although #394 currently has `oio` removed and loses outputs), and that my work had I think the same overall effect but was supplanted by Nico and Charlie's PRs. But maybe I shouldn't feel irked; I laid the groundwork and this is just the nature of collaboration, and Nico/Charlie are more senior here and have a better idea of how things should exactly be.

Also, I see that this PR does add more than #394, such as missing prefixes.